src/main/java/edu/greenblitz/robotName/subsystems/Gyros/GyroFactory.java

```diff
@@ -6,10 +6,10 @@ public class GyroFactory {
 
     public static IAngleMeasurementGyro create() {
         return switch (RobotConstants.ROBOT_TYPE) {
-            case ROBOT_NAME -> new PigeonGyro(GyroConstants.PigeonGyro.ID);
+            case ROBOT_NAME -> new Pigeon2Gyro(GyroConstants.Pigeon2Gyro.ID);
             case REPLAY -> new ReplayGyro();
+            case SIMULATION -> new SimulationGyro();
             default -> new SimulationGyro();
```

Comment:
you dont need the default if you answer all the enum cases

Reply:
רז רצה את זה

Reply:
בכל מקרה, אם יש שני קייסים שמחזירים אותו דבר כדאי לאחד אותם